docs/examples/go-ipfs-as-a-library/go.mod

```diff
@@ -3,11 +3,13 @@ module github.com/ipfs/go-ipfs/examples/go-ipfs-as-a-library
 go 1.14
 
 require (
-	github.com/ipfs/go-ipfs v0.5.0
-	github.com/ipfs/go-ipfs-config v0.5.3
+	github.com/ipfs/go-ipfs v0.7.0
+	github.com/ipfs/go-ipfs-config v0.9.0
 	github.com/ipfs/go-ipfs-files v0.0.8
-	github.com/ipfs/interface-go-ipfs-core v0.2.7
-	github.com/libp2p/go-libp2p-core v0.5.2
-	github.com/libp2p/go-libp2p-peerstore v0.2.3
-	github.com/multiformats/go-multiaddr v0.2.1
+	github.com/ipfs/interface-go-ipfs-core v0.4.0
+	github.com/libp2p/go-libp2p-core v0.6.0
+	github.com/libp2p/go-libp2p-peerstore v0.2.6
+	github.com/multiformats/go-multiaddr v0.2.2
 )
+
+replace github.com/ipfs/go-ipfs => ./../../..
```

Comment:
Interesting. Obviously we need this for now (i.e. until the go-ipfs 0.7.0 release) in order to get things to work, but should we just keep this replace directive here since it's an example?

Reply:
In e.g. go-car it is permanent: https://github.com/ipld/go-car/blob/v0.1.0/car/go.mod#L11
I don't mind either way, we can even remove it right now... ( there are no tests against it )